test/test_test.py

```diff
@@ -492,11 +492,39 @@ def test_reciprocal_blades(self, g):
                     assert ga.scalar_product(b1, rb2).simplify() == S.Zero
 
     def test_metric_collect(self):
-        ga = Ga('e*1|2', g=[1, 1, 1])
+        ga = Ga('e*1|2', g=[1, 1])
```

Comment:
So we failed to catch incompatible size of g? Should we raise exception here?

Reply:
I'm not yet sure if this is a feature or not.